api/src/main/java/org/apache/iceberg/PartitionSpec.java

```diff
@@ -48,23 +48,24 @@
  * represented by a named {@link PartitionField}.
  */
 public class PartitionSpec implements Serializable {
-  // start assigning IDs for partition fields at 1000
-  private static final int PARTITION_DATA_ID_START = 1000;
 
+  // start assigning IDs for partition fields at 1000
+  public static final int PARTITION_DATA_ID_START = 1000;
```

Comment:
This doesn't look like a real change. Probably caused by changing whitespace around these lines?